[Stream] Add changelog for live bandwidth estimates (#6616)
cloudflare/cloudflare-docs: content/stream/changelog.md

# Changelog

## 2022-10-20

### More accurate bandwidth estimates for live video playback

When playing live video, Cloudflare Stream now provides significantly more accurate estimates of the bandwidth needs of each quality level to client video players. This ensures that live video plays at the highest quality that viewers have adequate bandwidth to play.

> As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide the most accurate estimate of the bandwidth requirements in the HLS manifest.

Reviewer: > HLS manifest

This work applies for DASH manifests as well. I think we should we also mention it so folks don't get confused.

> As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide the most accurate estimate of the bandwidth requirements in the HLS manifest.

Author: ```suggestion
As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide the most accurate estimate of the bandwidth requirements in HLS (`.m3u8`) and DASH (`.mpd`) manifests.
```

For more on creating and managing live inputs, get started with the [docs](/stream/stream-live/).

## 2022-10-20

### More accurate bandwidth estimates for live video playback

When playing live video, Cloudflare Stream now provides significantly more accurate estimates of the bandwidth needs of each quality level to client video players. This ensures that live video plays at the highest quality that viewers have adequate bandwidth to play.

> As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide the most accurate estimate of the bandwidth requirements in HLS (`.m3u8`) and DASH (`.mpd`) manifests.

Author: ```suggestion
As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide an estimate of the bandwidth requirements of each quality level the in HLS (`.m3u8`) and DASH (`.mpd`) manifests.
```
